<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MBSWtable_Form</class>
 <widget class="QWidget" name="MBSWtable_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>475</width>
    <height>200</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Measuring Blocks</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="QTableWidget" name="selectedMBsSWs_tableWidget">
     <property name="minimumSize">
      <size>
       <width>382</width>
       <height>123</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ContiguousSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>Type:</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Title:</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Min. Value:</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Current Value:</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Max. Value:</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Unit:</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>8</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="mbswmovedown_pushButton">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>34</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/FreeSSM.qrc">
         <normaloff>:/icons/chrystal/32x32/1downarrow.png</normaloff>:/icons/chrystal/32x32/1downarrow.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mbswmoveup_pushButton">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>34</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/FreeSSM.qrc">
         <normaloff>:/icons/chrystal/32x32/1uparrow.png</normaloff>:/icons/chrystal/32x32/1uparrow.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="showMin_pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>34</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/FreeSSM.qrc">
         <normaloff>:/icons/freessm/24x24/show-min-value.png</normaloff>:/icons/freessm/24x24/show-min-value.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="showMax_pushButton">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>34</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/FreeSSM.qrc">
         <normaloff>:/icons/freessm/24x24/show-max-value.png</normaloff>:/icons/freessm/24x24/show-max-value.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetMinMax_pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>34</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/FreeSSM.qrc">
         <normaloff>:/icons/freessm/24x24/reset-min-max.png</normaloff>:/icons/freessm/24x24/reset-min-max.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/FreeSSM.qrc"/>
 </resources>
 <connections/>
</ui>
